addons/binding/org.openhab.binding.airvisualnode/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="airvisualnode"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="avnode">
+		<label>AirVisual Node</label>
+		<description>AirVisual Node air quality monitor</description>
+
+		<!-- Channel groups -->
+
+		<channel-groups>
+			<channel-group id="measurements" typeId="Measurements" />
+			<channel-group id="status" typeId="Status" />
+		</channel-groups>
+
+		<!-- Configuration parameters -->
+
+		<config-description>
+			<!-- Required parameters -->
+			<parameter name="address" type="text" required="true">
+				<context>network_address</context>
+				<label>Node Network Address</label>
+				<description>Node network address</description>
+				<required>true</required>
```

Comment:
You do not need this node, you have set this already via `required="true"` on the parameter node.